{"name": "test_AddOrg_StdPlan", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <Xero_Test.test_xero.Test_Xero object at 0x0000021CFBD5A550>, test_SetUp = None\n\n    @pytest.fixture()\n    def test_LoginToXero(self,test_SetUp):\n    \n        driver.get(\"https://login.xero.com/\")\n>       Reusable_Methods.compare(\"Login | Xero Accounting Software\", driver.title, \"Login Page\")\n\nXero_Test\\test_xero.py:92: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ntext1 = 'Login | Xero Accounting Software', text2 = 'Xero | Dashboard | HubUnique', element = 'Login Page'\n\n    @staticmethod\n    def compare(text1, text2, element):\n        if (text1 == text2 or (text1 in text2) or (text2 in text1)):\n            print(element, \" is displayed\")\n            assert True\n        else:\n    \n            print(element, \" is not displayed\")\n>           assert False\nE           AssertionError\n\nXero_Test\\ReusableMethods.py:15: AssertionError"}, "start": 1603986578749, "stop": 1603986578749, "uuid": "182938df-74eb-4c84-a310-5cc5e89417d5", "historyId": "7803f153634c0fa2b97f6cb762cf0e73", "testCaseId": "74ba57e3f1dd9b46bb9f46af7f654e71", "fullName": "Xero_Test.test_xero.Test_Xero#test_AddOrg_StdPlan", "labels": [{"name": "parentSuite", "value": "Xero_Test"}, {"name": "suite", "value": "test_xero"}, {"name": "subSuite", "value": "Test_Xero"}, {"name": "host", "value": "DESKTOP-I13U05B"}, {"name": "thread", "value": "17868-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Xero_Test.test_xero"}]}